docs/topics/mta-review-reports.adoc

```diff
@@ -0,0 +1,260 @@
+* A detailed description of the issue. This description outlines the problem, provides any known solutions, and references supporting documentation regarding either the issue or resolution.
+* A direct link, entitled *Show Rule*, to the rule that generated the issue.
+
+.Expanded issue
+image::3-5-expanded-issue.png[Expanded rule in the issues report]
+
+Issues are sorted into four categories by default. Information on these categories is available at ask Category.
+
+=== Application details report
+
+Access this report from the dashboard by clicking the *Application Details* link.
+
+The report lists the story points, the Java incidents by package, and a count of the occurrences of the technologies found in the application. Next is a display of application messages generated during the migration process. Finally, there is a breakdown of this information for each archive analyzed during the process.
+
+.Application Details report
+image::3-6-application-details-report.png[Application Details report]
+
+Expand the `jee-example-app-1.0.0.ear/jee-example-services.jar` to review the story points, Java incidents by package, and a count of the occurrences of the technologies found in this archive. This summary begins with a total of the story points assigned to its migration, followed by a table detailing the changes required for each file in the archive. The report contains the following columns.
+
+[cols="1,3", options="header"]
+|====
+| Column Name
+| Description
+
+| Name
+| The name of the file being analyzed.
+
+| Technology
+| The type of file being analyzed, for example, *Decompiled Java File* or *Properties*.
+
+| Issues
+| Warnings about areas of code that need review or changes.
+
+| Story Points
+a| Level of effort required to migrate the file.
+|====
+
+Note that if an archive is duplicated several times in an application, it will be listed just once in the report and will be tagged with `[Included multiple times]`.
+
+.Duplicate archive in an application
+image::3-7-duplicate-archive-in-application.png[Duplicate archive]
+
+The story points for archives that are duplicated within an application will be counted only once in the total story point count for that application.
+
+////
+[id="technology-report-application_{context}"]
+=== Technologies report
+
+Access this report from the dashboard by clicking the *Technologies* link.
+
+The report lists the occurrences of technologies, grouped by function, in the analyzed application. It is an overview of the technologies found in the application, and is designed to assist users in quickly understanding each application's purpose.
+
+The image below shows the technologies used in the `jee-example-app`.
+
+.Technologies in an application
+image::3-8-technologies-in-application.png[Technology report Application view]
+
+=== Transactions report

```

Comment:
Remove this entire section 
"=== Transactions report"

Reply:
Please search for "transaction report " and remove all occurrences.

Reply:
Done [commit](eb43b6e532d4bca68247b315697b7a1456e6adc3)